honeycomb/honeycomb.go

```diff
@@ -46,6 +46,9 @@ type Config struct {
 	// trouble getting the beeline to work, set this to true in a dev
 	// environment.
 	Debug bool
+	// APIHost is the hostname for the Honeycomb API server to which to send
+	// this event. default: https://api.honeycomb.io/
```

Comment:
These events? It's for more than one event. Otherwise people might think you can control sending event by event rather than by the whole exporter

Reply:
Good catch. will update my grammar :D